gallery.php

```diff
@@ -19,6 +19,21 @@
         <div class="table-title">
             <h1>Welcome In The Gallery</h1>
         </div>
-        <?php echo extractDataAndDisplay($result);?>
+        <nav class="navbar container">
+            <div>
+                <a href="index.php" class="">Back To Front Door</a>
+            </div>
+            <div>
+                <a href="form.php" class="">Add To You Collection</a>
+            </div>
+        </nav>
+        <?php echo extractDataAndDisplay($data);?>
+        <footer class="footer">
```

Comment:
Remove this class. You are applying this class to every footer, and only to footers, which makes it unnecessary.